backend/public-graph/graph/resolver.go

```diff
@@ -2671,6 +2673,7 @@ func (r *Resolver) ProcessPayload(ctx context.Context, sessionSecureID string, e
 				return e.Wrap(err, "error saving messages data")
 			}
 		} else {
+			// TODO - this code path is no longer hit.
```

Comment:
remove dead code?

Reply:
Was mostly leaving this as a comment to follow the logic. I can sync up with @mayberryzane and we can remove this in a follow up if he feels comfortable removing it now.